htdocs/cron/class/cronjob.class.php

```diff
@@ -1021,7 +1021,7 @@ function run_jobs($userlogin)
 				$result = call_user_func_array($this->methodename, $params_arr);
 			}
 
-			if ($result === false || $result != 0)
+			if ($result === false || $result < 0)
```

Comment:
Because some systems/os are not able to manage negative numbers for return on command line, the rule is that for cron method, 0 means ok, other means ko. To != 0 looks good to me.